package.json

```diff
@@ -15,7 +15,7 @@
   "license": "MIT",
   "devDependencies": {
     "pull-test": "^1.2.3",
-    "scuttlebot": "^9.4.6",
+    "scuttlebot": "^10.0.5",
```

Comment:
probably shouldn't have updated this. Haven't run tests. But it does speed up install D: